examples/user_guide/Network_Graphs.ipynb

```diff
@@ -0,0 +1,335 @@
+   "source": [
+    "As a final example let's look at a slightly larger graph. We will load a dataset of a Facebook network consisting a number of friendship groups identified by their ``'circle'``. We will load the edge and node data using pandas and then color each node by their friendship group using many of the things we learned above."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%opts Nodes Graph [width=800 height=800 xaxis=None yaxis=None tools=['hover']]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%opts Graph [color_index='circle']\n",
+    "%%opts Graph (node_size=10 edge_line_width=1)\n",
+    "colors = ['#000000']+hv.Cycle('Category20').values\n",
+    "edges_df = pd.read_csv('../assets/fb_edges.csv')\n",
+    "fb_nodes = hv.Nodes(pd.read_csv('../assets/fb_nodes.csv')).sort()\n",
+    "fb_graph = hv.Graph((edges_df, fb_nodes), label='Facebook Circles')\n",
+    "fb_graph = fb_graph.redim.range(x=(-0.05, 1.05), y=(-0.05, 1.05)).opts(style=dict(cmap=colors))\n",
+    "fb_graph"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Bundling graphs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The datashader library provides algorithms for bundling the edges of a graph and HoloViews provides convenient wrappers around the libraries."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from holoviews.operation.datashader import datashade, bundle_graph\n",
+    "bundled = bundle_graph(fb_graph)\n",
+    "bundled"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Datashading graphs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For graphs with a large number of edges we can datashade the paths and display the nodes separately. This loses some of the interactive features but will let you visualize quite large graphs:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%opts Nodes [color_index='circle'] (size=10 cmap=colors) Overlay [show_legend=False]\n",
+    "datashade(bundled, width=800, height=800) * bundled.nodes"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Applying selections"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Alternatively we can select the nodes and edges by an attribute that resides on either. In this case we will select the nodes and edges for a particular circle and then overlay just the selected part of the graph on the datashaded plot. In this way a smaller subgraph can be highlighted and the larger graph can be datashaded."
```

Comment:
This description sounds great, but I really couldn't make sense of the example; it's super hard to see what's datashaded and what isn't and what to make of it.  Again I think that's probably because this dataset isn't sufficiently large for there to be a difference between what bokeh alone can do and what datashader can.  Whatever it is, something needs to change for this to be a useful example.

Reply:
I can understand that visually it's not hugely compelling and a change in color schemes should help here, but is it really not clear conceptually? It's simply datashading everything and then overlaying part of the graph based on some criterion.

Reply:
The concept is clear to me, but it's not clear visually -- I can't tell which bits are overlaid and which bits are datashaded.  If I zoom quickly I can see that both are present, but I really can't visually get any idea which is which.

Reply:
The other issue is that bundling shifts the start and end points of the edges slightly (and sometimes significantly) so they don't line up with the nodes anymore. I'd expect bundling to leave the start and end points unaffected.

Reply:
@jbcrail ? I would also have expected bundling to leave nodes *exactly* where they are.

Reply:
@jbednar I would expect that also. A quick look at the code indicates at least two areas where the start and end points are resampled and modified.

@philippjfr Do you have an example? If so, could you put that into a datashader issue?

Reply:
@jbcrail Yes, I'll work up an example using the datashader API.